pytket/binders/circuit/main.cpp

```diff
@@ -517,6 +518,14 @@ PYBIND11_MODULE(circuit, m) {
           [](const Command &com) { return com.get_op_ptr()->free_symbols(); },
           ":return: set of symbolic parameters for the command");
 
+  py::class_<MetaOp, std::shared_ptr<MetaOp>, Op>(m, "MetaOp", "TODO")
+      .def(
+          py::init<OpType, op_signature_t, const std::string &>(),
+          "Construct from number of bits, bitwidths of inputs and outputs, "
+          "function name and module id.",
+          py::arg("type"), py::arg("signature"), py::arg("data"))
+      .def_property_readonly("data", &MetaOp::get_data, "Get data from MetaOp");
```

Comment:
Documentation looks wrong still.

Reply:
Oh, yes. Thank you. Had not seen that before.